app/tools/risk-survey/[resultId]/page.tsx

```diff
@@ -42,64 +75,220 @@ const RiskSurveyResults = async (props: Props) => {
+      <Flex flexDir={"column"}>
+        <Heading size={"lg"} marginBottom={4} marginTop={8}>
+          Want to Learn More?
+        </Heading>
+        <Text fontSize="md">
+          Checkout two blog posts that will give you a feel of what to expect
         </Text>
       </Flex>
-      <Card>
-        <Flex
-          flexDir={"column"}
-          justifyContent={"center"}
-          alignItems={"center"}
-          p={4}
-          w={"full"}
-          textAlign={"center"}
-        >
-          <Text
-            textColor={"black"}
-            opacity={0.5}
-            fontSize={{ base: "xs", md: "sm", lg: "md" }}
-          >
-            Summary
-          </Text>
-          <Text
-            color={"black"}
-            fontSize={{ base: "lg", md: "xl", lg: "2xl" }}
-            fontWeight={500}
+        <SimpleGrid columns={{ sm: 1, lg: 2 }} gap={{ sm: 4, lg: 6 }}>
+          <Stack gap={6}>
+            {toolDescriptions.map((tool, key) => (
+              <Tool key={key} tool={tool} />
+            ))}
+          </Stack>
+          <Card
+            variant={"whiteShadow"}
+            position={"relative"}
+            flexDir={"row"}
+            justifyContent={"space-between"}
+            style={{ display: "flex", flexDirection: "column", minHeight: 0 }}
           >
-            Your Risk Map
-          </Text>
-          <Text color={"black"} fontSize={"xl"} fontWeight={400}>
-            Visualise where your precieved risk overlaps with the community
-          </Text>
-          <SummaryChart userRisks={userRisks} means={means} />
-        </Flex>
-        <Flex w="full" justifyContent={"end"}>
-          <Link className="w-full md:w-fit" href={`/tools/rvol-calculator`}>
-            <Button
-              rightIcon={<FaCalculator />}
-              w="full"
-              colorScheme="picton-blue"
+            <Flex
+              flexDir="row"
+              justifyContent="space-between"
+              style={{ flex: 1 }}
+              gap={4}
             >
-              Risk Caclulator
-            </Button>
-          </Link>
-        </Flex>
-      </Card>
-
-      {userRisks.map((itm, key) => (
-        <InvestmentCard key={key} investment={investmentMap[itm.investmentId!]}>
-          <InvestmentChart
-            userRisk={itm.riskLevel}
-            investmentRvol={investmentMap[itm.investmentId!].rvol}
-            investmentId={itm.investmentId!}
-            investmentVotes={votes.filter(
-              (v) => v.investmentId === itm.investmentId
-            )}
-          />
-        </InvestmentCard>
-      ))}
-      <CopyUrlButton colorScheme="hollywood">Copy Result Link</CopyUrlButton>
+              <Flex flexDir={"column"} gap={4}>
+                <Heading size={"md"}>Blog Post</Heading>
+                <Text fontSize={"md"} color={"red"}>Link with specific Blog-post-slug, make the blog page first</Text>
+              </Flex>
+              <Flex minW={"30%"} justifyContent={"center"}>
```

Comment:
Be careful with setting minW = a percentage, how does this look on mobile?

Reply:
yeah good point will check